spec/tutum_spec.rb

```diff
@@ -2,51 +2,94 @@
 
 test_username = ENV['TUTUM_USERNAME']
 test_api_key = ENV['TUTUM_API_KEY']
+test_tutum_auth = "ApiKey #{test_username}:#{test_api_key}" #best way to handle this?
```

Comment:
I would do this as ENV['TUTUM_AUTH'] in line with the username and api key. This way we're actually testing comms works with both methods